### PR TITLE
Fix for #12692 Cloud CDN Singed Urls and Signed Cookies examples use time-zone unaware datetime objects

### LINKED ISSUE #12692: Cloud CDN Singed Urls and Signed Cookies examples use time-zone unaware datetime objects
The code snippets of Cloud CDN use [naive datetime objects](https://docs.python.org/3/library/datetime.html#aware-and-naive-objects) as expiration times and depend on deprecated [datetime.utcfromtimestamp()](https://docs.python.org/3/library/datetime.html#datetime.datetime.utcfromtimestamp) function. It is not apparent for the users that they are supposed to pass UTC expiration datetime.

If unnoticed by a user of the snippet, this may lead to generating signatures that are valid for shorter or longer then expected.

## In which file did you encounter the issue?
`cdn/snippets.py`

## Describe the issue
If a user create an expiration datetime that depends on local timezone, e.g. by adding expiration period to `datetime.now()`, the signature may be longer or shorter than expected, depending on the local-time zone, where the code was executed.

Instead, the code snippets should use time-zone aware datetime objects, to ensure that time-zone offsets are taken into account, when the signatures are calculated.

### PR DESCRIPTION
**Pull request by pawelkedzior-gl**
## Description

Fixes #12692

## Checklist
- [☑] I have followed [Sample Guidelines from AUTHORING_GUIDE.MD](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/AUTHORING_GUIDE.md)
- [☑] README is updated to include [all relevant information](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/AUTHORING_GUIDE.md#readme-file)
- [☑] **Tests** pass:   `nox -s py-3.9` (see [Test Environment Setup](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/AUTHORING_GUIDE.md#test-environment-setup))
```
pytest
snippets_test.py::test_sign_url PASSED                                   [ 16%]
snippets_test.py::test_sign_url_raise_exception_on_naive_expiration_datetime PASSED [ 33%]
snippets_test.py::test_sign_url_prefix PASSED                            [ 50%]
snippets_test.py::test_sign_url_prefix_raise_exception_on_naive_expiration_datetime PASSED [ 66%]
snippets_test.py::test_sign_cookie PASSED                                [ 83%]
snippets_test.py::test_sign_cookie_raise_exception_on_naive_expiration_datetime PASSED [100%]
```
- [?] **Lint** pass:   `nox -s lint` (see [Test Environment Setup](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/AUTHORING_GUIDE.md#test-environment-setup))
```
nox -s lint
nox > Failed to load Noxfile {redacted}/python-docs-samples/cdn/noxfile.py, no such file exists.
```
- [no] These samples need a new **API enabled** in testing projects to pass (let us know which ones)
- [no] These samples need a new/updated **env vars** in testing projects set to pass (let us know which ones)
- [no] This sample adds a new sample directory, and I updated the [CODEOWNERS file](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/.github/CODEOWNERS) with the codeowners for this sample
- [no] This sample adds a new **Product API**, and I updated the [Blunderbuss issue/PR auto-assigner](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/.github/blunderbuss.yml) with the codeowners for this sample
- [?] Please **merge** this PR for me once it is approved

### CODE REVIEW

File: cdn/snippets.py [hunk 1]
```diff
@@ -27,6 +27,7 @@
 import argparse
 import base64
 from datetime import datetime
```
Comment (iennae) [inline cdn/snippets.py:29]: nit: could you put this on one line 
```
from datetime import datetime, timezone 
```

**Comment by iennae** [comment]
@pawelkedzior-gl you might want to resubmit this PR as your first commit doesn't appear to be correctly associated to your Google configured GitHub account. Additionally, linting on the imports is what is breaking right now.

**Comment by pawelkedzior-gl** [comment]
@iennae - thank  you, yes, I will do that

**Comment by pawelkedzior-gl** [comment]
A new pull request:
https://github.com/GoogleCloudPlatform/python-docs-samples/pull/12704

File: cdn/snippets.py (hunk 1 above)
Comment (pawelkedzior-gl) [reply cdn/snippets.py:29]: Resolved in f3b87f0ef11defcb1efdd4b8d140981fa00406db of the new pull request

**Pull request closed without merging**